package.json

```diff
@@ -27,37 +27,55 @@
     "Google Cloud Vision API"
   ],
   "scripts": {
+    "test": "c8 mocha build/test",
+    "samples-test": "cd samples/ && npm link ../ && npm install && npm test && cd ../",
+    "lint": "gts fix && eslint --fix samples/*.js",
     "docs": "jsdoc -c .jsdoc.js",
-    "lint": "eslint '**/*.js'",
-    "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
-    "system-test": "mocha system-test/*.js smoke-test/*.js --timeout 600000",
+    "system-test": "mocha build/system-test",
+    "fix": "gts fix",
     "test-no-cover": "mocha test/*.js",
-    "test": "nyc mocha",
-    "fix": "eslint --fix '**/*.js'",
     "docs-test": "linkinator docs",
     "predocs-test": "npm run docs",
-    "prelint": "cd samples; npm link ../; npm i"
+    "prelint": "cd samples; npm link ../; npm i",
+    "clean": "gts clean",
+    "compile": "tsc -p . && cp -r protos build/",
+    "compile-protos": "compileProtos src",
+    "prepare": "npm run compile"
   },
   "dependencies": {
-    "@google-cloud/promisify": "^1.0.0",
-    "google-gax": "^1.7.5",
-    "is": "^3.2.1"
+    "@google-cloud/promisify": "^1.0.4",
+    "@types/is": "0.0.21",
+    "@types/sinon": "^7.5.1",
```

Comment:
The `@types/` dependencies should all be `devDependencies`

Reply:
done!